build.gradle

```diff
@@ -1,6 +1,6 @@
 plugins {
-    id 'java'
-    id 'application'
+    id "java"
+    id "application"
 }
```

Comment:
Why using double quotes for this and single quotes for the rest ?

Reply:
Fixed